CMakeLists.txt

```diff
@@ -354,20 +367,40 @@ if(SDL3IMAGE_ZLIB)
     endif()
 endif()
 
+if(SDL3IMAGE_DAV1D)
```

Comment:
Should this not be cover under the `if(SDL3IMAGE_AVIF_VENDORED)` block?

Reply:
You're right it would be equivalent to put it under the `if(SDL3IMAGE_AVIF_VENDORED)`, but I didn't do it for cosmatic reasons and consistency. The script also does this for zib.

Reply:
>  The script also does this for zib.

Then zlib handling should be changed too, IMO